TutorLizard.BusinessLogic/Data/DataAccess.cs

```diff
@@ -196,4 +198,55 @@ private int GetNewScheduleItemRequestID()
             return 0;
     }
     #endregion
+
+    private bool DoesTheUserIdExist(int id)
```

Comment:
This relies too heavily on assumption that we'll never have any gaps in ids. Any manual deletion could break it. Why not simple `Any()`?